controllers/agents/app/serviceclaim_controller.go

```diff
@@ -134,6 +137,15 @@ func (r *ServiceClaimReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	})
 	if err != nil {
 		if strings.Contains(err.Error(), "admission webhook \"vserviceclaim.kb.io\" denied the request") {
+			c := metav1.Condition{
+				LastTransitionTime: metav1.NewTime(time.Now()),
+				Type:               primazaiov1alpha1.ServiceBindingConditionReady,
+				Status:             metav1.ConditionFalse,
+				Reason:             "WebhookError",
+				Message:            "",
```

Comment:
I'd also propose to use a less technical Reason here

```suggestion
				Reason:             "ValidationError",
				Message:            err.Error(),
```

Reply:
The error object is not available. But I have updated the message. Please check.